member-service/src/main/kotlin/org/jeongmo/migration/member/domain/model/Member.kt

```
@@ -0,0 +1,19 @@
+class Member(
+    val id: Long?,
+    var username: String,
+    var password : String?,
+    var providerType : ProviderType,
+    var nickname : String,
+    var role : Role,
+    var deletedAt : LocalDateTime?,
+    createdAt: LocalDateTime? = null,
+    updatedAt: LocalDateTime? = null,
+): BaseDomain(createdAt, updatedAt) {
+}
```

detekt (1.23.8)

[warning] The class or object Member is empty.

(detekt.empty-blocks.EmptyClassBlock)